gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -116,8 +116,7 @@
                         <input name="maxSendKbps" id="maxSendKbps" class="form-control" type="number" ng-model="currentDevice.maxSendKbps" required="" aria-required="true" min="0"/>
                       </div>
                     </div>
-                    <p class="help-block" ng-if="!deviceEditor.maxSendKbps.$valid && deviceEditor.maxSendKbps.$dirty" translate>
-                      The rate limit must be a non-negative number (0: no limit) </p>
+                    <p class="help-block" ng-if="!deviceEditor.maxSendKbps.$valid && deviceEditor.maxSendKbps.$dirty" translate>The rate limit must be a non-negative number (0: no limit) </p>
```

Comment:
there is a space before `</p>`- I don't know if this is a problem for translation strings.

Reply:
I had the same comment, make sure its identical to the code in the other place.

Reply:
There are a few such spaces before closing tags in current device modal but not in folder modal. I will get rid of all just to be safe.

Reply:
Well to be exact there are no such spaces in code from master. I must have added them accidentally when I was fooling around with formatting. Anyway, they are gone.